doc/builders/images/portableservice.section.md

```diff
@@ -0,0 +1,81 @@
+# pkgs.portableService {#sec-pkgs-portableService}
+
+`pkgs.portableService` is a function to create _"portable service images"_,
```

Comment:
```suggestion
`pkgs.portableService` is a function to create _portable service images_,
```

I hadn't made a suggestion because I didn't know what you wanted to do here. Possibly also a misunderstanding, because I implicitly considered quotes a highlight as well...